hawkbit-core/src/main/java/org/eclipse/hawkbit/tenancy/configuration/validator/TenantConfigurationPollingDurationValidator.java

```diff
@@ -47,18 +45,19 @@ public void validate(final Object tenantConfigurationObject) {
 
         final Duration tenantConfigurationValue;
         try {
-            tenantConfigurationValue = durationHelper.formattedStringToDuration(tenantConfigurationString);
+            tenantConfigurationValue = DurationHelper.formattedStringToDuration(tenantConfigurationString);
         } catch (final DateTimeParseException ex) {
             throw new TenantConfigurationValidatorException(
                     String.format("The given configuration value is expected as a string in the format %s.",
-                            DurationHelper.DURATION_FORMAT));
+                            DurationHelper.DURATION_FORMAT),
+                    ex);
```

Comment:
Is think this is a false postive, because would we show the parsing expception to the user?